src/screens/Home/ResourcesItemScreen.js

```diff
@@ -0,0 +1,161 @@
+  const [body, setBody] = useState(null);
+  const [steps, setSteps] = useState(null);
+  const [subheader, setSubheader] = useState(null);
+  const [authorName, setAuthorName] = useState(null);
+
+  /* Screen state */
+  const [loading, setLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  /* Query data from firestore */
+  const getData = useCallback(async () => {
+    setLoading(true);
+
+    try {
+      /* Query data from resource item collection */
+      const resourceSnap = await firestore().collection('resource_items').doc(resourceID).get();
+      const resourceData = resourceSnap.data();
+
+      /* Query user data */
+      const authorSnapshot = firestore().collection('users').doc(resourceData.userID).get();
+      const authorSnap = await Promise.all([authorSnapshot]);
+
+      setAuthorName(authorSnap.length ? authorSnap[0].get('displayName') : null);
+      setTitle(resourceData.title);
+      setDescription(resourceData.description);
+      setType(resourceData.type);
+      setSubheader(resourceData.subheader);
+
+      /* Different behavior if resource is of type STEPS or FREEFORM */
+      if (type === 'STEPS') {
+        const stepsList = await firestore().collection('resource_items').doc(resourceID).collection('steps')
+          .get();
+        const stepsData = stepsList.docs.map((doc) => ({ ...doc.data(), id: doc.id }));
+
+        const sortedStepsData = stepsData.sort((a, b) => {
+          if (a.stepNumber > b.stepNumber) {
+            return 1;
+          }
+          if (a.stepNumber < b.stepNumber) {
+            return -1;
+          }
+          return 0;
+        });
+
+        setSteps(sortedStepsData.map((step) => ({
+          ...(<ResourceStep
+            stepNumber={step.stepNumber}
+            title={step.title}
+            description={step.body}
+          />),
+          key: step.id,
+        })));
+      } else if (type === 'FREEFORM') {
+        setBody(resourceData.body);
+      }
+
+      setLoading(false);
+    } catch (error) {
+      setErrorMessage(error.message);
+      setLoading(false);
+    }
+  }, [resourceID, type]);
+
+  useEffect(() => {
+    getData();
+  }, [getData]);
+
+  return (
+    <View style={resourcesStyles.container}>
+      <ScrollView style={resourcesStyles.container}>
+        <View style={resourcesStyles.itemContainer}>
+          {errorMessage && <Text>{errorMessage}</Text>}
+          {loading && <ActivityIndicator />}
+          <Title>{title}</Title>
+          <Paragraph>{description}</Paragraph>
+          <Text>{'\n'}</Text>
+          {steps != null
+            ? (
+              <List.Section>
+                <List.Subheader>{subheader}</List.Subheader>
+                {steps}
+              </List.Section>
+            )
+            : (
+              <View>
+                <Subheading>{subheader}</Subheading>
+                <Paragraph>{body}</Paragraph>
+              </View>
+            )}
```

Comment:
`steps` is initially null, so the second part will always render when the component is mounted. Change to
```
{steps && ( ... )}
{body && ( ... )}
```